Build Hangs in V4

**Describe the bug**
I followed the expo-router instructions for v4.
When including the css file in _layout.css the build hangs

**To Reproduce**
Steps to reproduce the behavior:
1. Clone this repo https://github.com/revolution42/nativewind-demo
2. Run yarn start
3. Open in web mode (W)
4. See error

**Screenshots**
![image](https://github.com/marklawlor/nativewind/assets/322514/7a6407fd-6fc8-47cf-9bd4-435480ea4168)
Build will stay like this

Details about your system:

   - what system are you using

Windows 11

   - what command line are you using

I am using powershell and yarn, i also tried clearing it all and using npm

   - do you have the same issue on a different computer

I spun up a azure windows computer and tried ubuntu in WSL and I got this
error message for both

> Error: Cannot find module '../lightningcss.linux-x64-gnu.node'
>
> Require stack:
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/react-native-css-interop/node_modules/lightningcss/node/index.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/react-native-css-interop/dist/css-to-rn/index.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/react-native-css-interop/css-to-rn/index.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/nativewind/dist/metro/tailwind-cli.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/nativewind/dist/metro/index.js
>
> - /home/shannon/Workspace/nativewind-demo/metro.config.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/import-fresh/index.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/cosmiconfig/dist/loaders.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/cosmiconfig/dist/createExplorer.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/cosmiconfig/dist/index.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/metro-config/src/loadConfig.js
>
> -
>> /home/shannon/Workspace/nativewind-demo/node_modules/metro-config/src/index.js
>
> -
>> ***@***.***/metro-config/build/traveling/metro-config.js
>
> -
>> ***@***.***/metro-config/build/ExpoMetroConfig.js
>
> -
>> ***@***.***/cli/build/src/start/server/metro/resolveFromProject.js
>
> -
>> ***@***.***/cli/build/src/utils/analytics/getMetroProperties.js
>
> -
>> ***@***.***/cli/build/src/start/server/metro/instantiateMetro.js
>
> -
>> ***@***.***/cli/build/src/start/server/metro/MetroBundlerDevServer.js
>
> -
>> ***@***.***/cli/build/src/start/server/DevServerManager.js
>
> -
>> ***@***.***/cli/build/src/start/startAsync.js
>
> -
>> ***@***.***/cli/build/src/start/index.js
>
> -
>> ***@***.***/cli/build/bin/cli
>
> - /home/shannon/Workspace/nativewind-demo/node_modules/expo/bin/cli
>
>

Details about the type of build

   - do production builds also free

Expo production mode freezes aswell
npx expo export -p web also freezes

   - if you using Expo Go, does it free when making a development build

Expo Go and Android with emulator work it looks like it is something to do
with the web build



On Sat, 4 Nov 2023 at 14:23, Mark Lawlor ***@***.***> wrote:

> Ok, I need more information and some more troubleshooting from you.
>
> Details about your system:
>
>    - what system are you using
>    - what command line are you using
>    - do you have the same issue on a different computer
>
> Details about the type of build
>
>    - do production builds also free
>    -if you using Expo Go, does it free when making a development build
>
> I'm sure there is something fixable here, but we need to first discover
> the cause.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/marklawlor/nativewind/issues/610#issuecomment-1793358515>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACOXUUVCZRFMJU5M5WET3TYCXNONAVCNFSM6AAAAAA647FU4OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOJTGM2TQNJRGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

--- a/packages/nativewind/src/metro/transformer.ts
+++ b/packages/nativewind/src/metro/transformer.ts
@@ -48,7 +48,7 @@ StyleSheet.registerCompiled(JSON.parse('${config.nativewind.parsedOutput}'));`,
         config,
         projectRoot,
         filename,
-        Buffer.from(`require('${config.nativewind.outputPath}');`, "utf8"),
+        Buffer.from(`require('${config.nativewind.outputPath.replace(/\\/g, '\\\\')}');`, "utf8"),
         options,
       );
     } else {

